RecoLocalMuon/RPCRecHit/src/RPCRecHitStandardAlgo.cc

```diff
@@ -24,18 +25,32 @@ bool RPCRecHitStandardAlgo::compute(const RPCRoll& roll,
   const float fstrip = (roll.centreOfStrip(cluster.firstStrip())).x();
   const float lstrip = (roll.centreOfStrip(cluster.lastStrip())).x();
   const float centreOfCluster = (fstrip + lstrip)/2;
+  const double y = cluster.hasY() ? cluster.y() : 0;
+  Point = LocalPoint(centreOfCluster, y, 0);
```

Comment:
OK, imagine the roll is trapezoidal with bottom 1/2-width 20, middle half-width 40, and top half-width 60 and we are on the last strip (one-strip cluster); and y of the top is +10 vs y of the bottom -10.

what are the possible values of measured y? 
Depending on y the point x coordinate will be between 20 and 60.
Are you saying this is not possible and no matter what value of y is, the x of the point here will still be 40?
Maybe I misunderstand the code and in reality centreOfCluster is actually going to be dependent on y in this example.

Please clarify.
Thank you.